tests/integration/conftest.py

```diff
@@ -234,4 +234,9 @@ def event_loop():
     """
     loop = asyncio.get_event_loop_policy().new_event_loop()
     yield loop
+    # gracefully exit async generators
+    loop.run_until_complete(loop.shutdown_asyncgens())
```

Comment:
#TIL 

First time I see this function, so had to google it: https://docs.python.org/3/library/asyncio-eventloop.html#asyncio.loop.shutdown_asyncgens :+1:

Reply:
Note that I added this just because it seemed like a good idea, commenting out this line has no actual effect.